### PR TITLE
Added eap local account check in Freeradius

### PR DESCRIPTION
**Pull request by fdurand**
Added local account created from teh admin GUI

### CODE REVIEW

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 1]
```diff
@@ -1082,16 +1082,17 @@ authorize {
...
 #                MS-CHAP-Use-NTLM-Auth := No
 #            }
+## Check temporary_password table with email and password for a sponsor registration
 #            pfguest
 #            if (fail || notfound) {
 ## Check temporary_password table with email and password for a guest registration
 #                pfsponsor
 #                if (fail || notfound) {
-## Check temporary_password table with email and password for a sponsor registration
+## Check activation table with telephone number and pin code
```
Comment (julsemaan) [inline docs/PacketFence_Administration_Guide.asciidoc]: phone number would be better
and I think it should be PIN code

File: raddb/sites-available/packetfence-tunnel [hunk 2]
```diff
@@ -11,20 +11,25 @@ authorize {
...
 ## Check temporary_password table with email and password for a sponsor registration
-#                    pfsms
+#                pfguest
+#                if (fail || notfound) {
+## Check temporary_password table with email and password for a guest registration
+#                    pfsponsor
 #                    if (fail || notfound) {
-#                        update control {
-#                           MS-CHAP-Use-NTLM-Auth := Yes
+## Check activation table with telephone number and pin code
```
Comment (julsemaan) [inline raddb/sites-available/packetfence-tunnel]: same would apply here

**Comment by julsemaan** [comment]
needs news entry

**Comment by fdurand** [comment]
News entry already there

**Comment by julsemaan** [comment]
I can't see it in this PR. Is it in the devel NEWS file ?

**Comment by fdurand** [comment]
In NEWS.asciidoc:
Added local account authentication for EAP

**Pull request merged**